metadata.rb

```diff
@@ -10,6 +10,6 @@
   openstack-common openstack-compute openstack-dashboard openstack-identity
   openstack-image openstack-network openstack-object-storage
   openstack-ops-database openstack-ops-messaging openstack-orchestration
-  openstack-telemetry }.each do |cb|
+  openstack-telemetry yum-fedora}.each do |cb|
```

Comment:
Hmm. I'm not sure off the top of my head if this will affect the cookbook attributes loading precedence. Have you run it with the `base_managed` role to confirm it doesn't break the update repo again?